app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -127,6 +128,7 @@ export default function ProfileTagInput({
   id,
   allowCsv = true,
   className,
+  renderLabel,
```

Comment:
Why is this prop needed? Doesn't look like it's used anywhere at the moment?

Reply:
Weird, this prop seems to be already removed here.

Reply:
yes, can be removed.